doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md

```diff
@@ -155,34 +155,134 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
+    * CYGNUS_CKAN_ENABLE_NAME_MAPPINGS: true if name mappings are enabled for this sink, false otherwise.
+    * CYGNUS_CKAN_DATA_MODEL: select the data_model: dm-by-service-path or dm-by-entity.
+    * CYGNUS_CKAN_ATTR_PERSISTENCE: how the attributes are stored, either per row either per column (row, column).
+    * CYGNUS_CKAN_ORION_URL: Orion URL used to compose the resource URL with the convenience operation URL to query it
+    * CYGNUS_CKAN_BATCH_SIZE: number of notifications to be included within a processing batch.
+    * CYGNUS_CKAN_BATCH_TIMEOUT: timeout for batch accumulation.
+    * CYGNUS_CKAN_BATCH_TTL: number of retries upon persistence error.
+    * CYGNUS_CKAN_BACKEND_MAX_CONNS: maximum number of Http connections to CKAN backend
+#cygnus-ngsi.sinks.ckan-sink.backend.max_conns = 500
+    * CYGNUS_CKAN_BACKEND_MAX_CONSS_PER_ROUTE: maximum number of Http connections per route to CKAN backend
+
 * HDFS:
     * It only works for building historical context data in HDFS.
     * The endpoint for HDFS is `iot-hdfs` but can be changed through the CYGNUS_HDFS_HOST environment variable.
     * The port for HDFS is `50070` but can be changed through the CYGNUS_HDFS_PORT environment variable.
     * The user for HDFS is `hdfs` but can be changed through the CYGNUS_HDFS_USER environment variable.
     * The token for HDFS is empty but can be set through the CYGNUS_HDFS_TOKEN environment variable
+    * CYGNUS_HDFS_ENABLE_ENCODING: true applies the new encoding, false applies the old encoding.
+    * CYGNUS_HDFS_ENABLE_GROUPING: true if the grouping feature is enabled for this sink, false otherwise.
+    * CYGNUS_HDFS_ENABLE_NAME_MAPPINGS: true if name mappings are enabled for this sink, false otherwise.
+    * CYGNUS_HDFS_ENABLE_LOWERCASE: true if lower case is wanted to forced in all the element names, false otherwise.
+    * CYGNUS_HDFS_DATA_MODEL: select the data_model: dm-by-service-path or dm-by-entity.
+    * CYGNUS_HDFS_FILE_FORMAT: how the attributes are stored, available formats are json-row, json-column, csv-row and csv-column
+    * CYGNUS_HDFS_BACKEND_IMPL: rest if the interaction with HDFS will be WebHDFS/HttpFS-based, binary if based on the Hadoop API
+    * CYGNUS_HDFS_BACKEND_MAX_CONNS: maximum number of Http connections to HDFS backend
+    * CYGNUS_HDFS_BACKEND_MAX_CONNS_PER_ROUTE: maximum number of Http connections per route to HDFS backend
+    * CYGNUS_HDFS_PASSWORD: password for the above username; this is only required for Hive authentication
+    * CYGNUS_HDFS_SERVICE_AS_NAMESPACE: true if the notified fiware-service (or the default one, if no one is notified) is used as the HDFS namespace, false otherwis
+    * CYGNUS_HDFS_BATCH_SIZE:  number of notifications to be included within a processing batch.
+    * CYGNUS_HDFS_BATCH_TIMEOUT: timeout for batch accumulation.
+    * CYGNUS_HDFS_BATCH_TTL: number of retries upon persistence error.
+    * CYGNUS_HDFS_BATCH_RETRY_INTERVALS
+    * CYGNUS_HDFS_HIVE: Hive enabling
```

Comment:
Using the same style in other parts:

true enables Hive, false disabled it
true enables Kerberos-basded authentication, false disables it